checks/evaluation/code_review.go

```diff
@@ -130,5 +93,17 @@ func reviewScoreForChangeset(changeset *checker.Changeset) (score reviewScore) {
 		}
 	}
 
+	plat := changeset.ReviewPlatform
+	if plat == "" {
+		plat = checker.ReviewPlatformUnknown
+	}
```

Comment:
Would it be better to set the platform to known here:
https://github.com/ossf/scorecard/blob/332472d4d67be05f151151244ed3aa3da8d3f975/checks/raw/code_review.go#L127-L145

Instead of adding the logic here?